Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -50,7 +50,7 @@ extension Snapshotting where Value == URLRequest, Format == String {
     switch httpMethod {
     case "GET": break
     case "HEAD": components.append("--head")
-    default: components.append("--request \(httpMethod)")
+    default: components.append("--request \(httpMethod)")```
```

Comment:
Oops, accidentally left this in when commenting...

```suggestion
    default: components.append("--request \(httpMethod)")
```